### DISCUSSION TITLE
Upgrade from SO 2.3.90 to 2.3.91

### DISCUSSION THREAD

**Discussion by adipetrisoaia** [Security-Onion-Solutions/securityonion#6903] [Unsupported Versions] [answered]
Hello,

I encounter a problem during upgrade from 90 to 91 version.

the issue is as follows:

![Inkedrawww_LI](https://user-images.githubusercontent.com/85494455/149963009-61aec770-bc93-466d-b051-c9fe719650eb.jpg)

There seems to be a typo, see the masterfully highlighted link from above.
 Someone else encountered this issue? Please assist, thanks

Adrian

**Comment by dougburks** [comment, marked as answer]
The typo is just in the error message itself and this has already been fixed for 2.3.100:
https://github.com/Security-Onion-Solutions/securityonion/commit/7d1f9c51e8fdb04e4787285e63f2b52759b02a43#diff-32d6509a40a77b74ab51a4511bf3981b44c32f1c695032c0d9315a26a560fee5

You'll want to figure out why you can't connect to raw.githubusercontent.com.

**Reply by adipetrisoaia** [reply]
This is the message I receive when I wget from the machine to githhub.
![github](https://user-images.githubusercontent.com/85494455/150134129-0f785d24-670c-42e1-b2a2-8ce88944ba4d.PNG)

**Reply by dougburks** [reply]
You may need to examine `index.html.1` to see if it actually looks like the expected content from raw.githubusercontent.com. Your screenshot shows `Proxy request sent` so it's possible that `index.html.1` shows an error from your proxy server.

**Reply by adipetrisoaia** [reply]
used the "-i" variable as the update process was not reading the enviroment settings we set. the update was succesful to 91 version.

Thanks for your replys.
